### DISCUSSION TITLE
Force to take External Converter

### DISCUSSION THREAD

**Discussion by Danit2** [Koenkk/zigbee2mqtt#9358] [Devices]
Hi to all.

Because one device is not working correct i have made e new "external converter" for the device.

How can i force, that Z2M take the file from the external converter instead of the internal?

Everytime when i repair the device, the programm take the internal converter. 

Or can you tell me, where i finde the device libary? I use HASSOS with the officially add-on.

Thanks for any help.

**Comment by Koenkk** [comment]
This is a bug in z2m 1.21.2, and is fixed in the dev (external definition is always picked over internal). (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

**Comment by Danit2** [comment]
Thanks @Koenkk 

I have changed to the dev and it works as expected.
